### ISSUE TITLE
Empty config dialog

### ISSUE THREAD

**Issue by kaorahi** [featurecat/lizzie#616]
On my linux machine, I get an empty window by the menu [Settings] ==> [Engine]. It is closed by Enter key, though no buttons are shown on the window.

If I comment out "setType(Type.POPUP);" in ConfigDialog() in ConfigDialog.java, the contents of the window seem to be shown correctly. If I replace the line with "setAlwaysOnTop(true);", it still works well.

I know nothing on swing and have no idea...

~~~~
$ git log --oneline -n 1
7ce899c (HEAD -> master, upstream/master) Add the Wiki link. (#610)
~~~~

**Comment by featurecat** [comment]
@zsalch could you please investigate this?

**Comment by hanysz** [comment]
Same issue here (on Ubuntu 18.04 and Java 10.0.2), and the same fix works.  I'm also finding that the GTP console is an empty window (grey rectangle), looks like the same issue except that I can't see a "setType()" line in GtpConsolePane.java.

**Comment by hanysz** [comment]
OK, GTP issue fixed for me, to some extent it was user error :-)  I found a menu item View -> Panel -> GTP console which is intended to toggle the console between visible and hidden.  On startup ,the GTP console is disabled but for some reason still visible as a grey rectangle.  Toggling once makes it functional.  Toggling again makes it disappear.

**Comment by etibla** [comment]
It seems I have exactly the same errors. 

I try to run the lizzie.jar release for mac/linux, version 0.7
I tried with open JDK and the official java 8 oracle

Impossible to open Engine window or any other window in this menu), instead I have a blank window. I can have this blank window on top disappear using "enter" but my console shows `Exception in thread "Thread-1" java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
	at java.base/java.util.Objects.checkIndex(Objects.java:372)
	at java.base/java.util.ArrayList.get(ArrayList.java:458)
	at featurecat.lizzie.analysis.EngineManager.updateEngine(EngineManager.java:230)
	at featurecat.lizzie.analysis.EngineManager.lambda$null$6(EngineManager.java:175)
	at java.base/java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:104)
	at java.base/java.util.stream.IntPipeline$Head.forEach(IntPipeline.java:593)
	at featurecat.lizzie.analysis.EngineManager.lambda$null$7(EngineManager.java:170)
	at java.base/java.util.Optional.ifPresent(Optional.java:183)
	at featurecat.lizzie.analysis.EngineManager.lambda$refresh$8(EngineManager.java:167)
	at java.base/java.lang.Thread.run(Thread.java:834)
`

I also have the same console problem as above and don't think the issue is fixed. toggling it and untoggling it might be more of a workaround :D

**Comment by kaorahi** [comment]
fixed by #642

**Comment by etibla** [comment]
Works for me, thank you!
